README.md

```diff
@@ -163,9 +164,10 @@ JavaScript applications. It runs both in Node.js and browser.
 
 * * *
 
-<a name="new_ApifyClient_new"></a>
+<a name="ApifyClient"></a>
```

Comment:
I really can't figure out how to do this one, since 'new' isn't included in the props

![Screenshot 2020-12-30 at 19 14 01](https://user-images.githubusercontent.com/38246758/103372763-34d71d80-4ad3-11eb-813d-cfd1745ddbb3.png)